logging-log4j/pom.xml

```diff
@@ -59,7 +59,7 @@
     </developers>
     <properties>
         <foundation-common-base-version>1.0.1-5</foundation-common-base-version>
-        <logging-api-version>1.1.0-1</logging-api-version>
+        <logging-api-version>1.2.0-1</logging-api-version>
```

Comment:
you'll also need to update the logging-api dependency yet again after logging-api release and only then release logging-log4j pointing to latest.